www/includes/easyparliament/templates/html/misc/highlighted_2024.php

```diff
@@ -0,0 +1,95 @@
+<?php $category_ids = $array = [
+    '2'   => 'Donations and other support (including loans) for activities as an MP',
+    '1.2' => 'Employment and earnings - Ongoing paid employment',
+    '1.1' => 'Employment and earnings - Ad hoc payments',
+    '4'   => 'Visits outside the UK',
+    '3'   => 'Gifts, benefits and hospitality from UK sources',
+    '8'   => 'Miscellaneous',
+];
+; ?>
+
+<div class="full-page static-page toc-page">
+    <div class="full-page__row">
+        <div class="toc-page__col">
+            <div class="toc js-toc">
+            <ul>
+                <li><a href="#intro">Introduction</a></li>
+                <?php foreach ($category_ids as $category_id => $category_name) { ?>
+                    <li><a href="#category-<?= $category_id ?>"><?= $category_name ?></a></li>
+                <?php }; ?>
+            </ul>
+        </div>
+
+        </div>
+        <div class="toc-page__col">
+
+            <div class="panel">
+                <h1>Highlighted interests</h1>
+
+                <a name="intro"></a>
+                <p>As part of our 2024/25 WhoFundsThem project, we are highlighting declared interests that we feel merit being more visible, based on known public attitudes towards certain industries.</p>
+                <p>You can read more about our research and choices in the accompanying <a href="http://research.mysociety.org/html/beyond-transparency/">Beyond Transparency report</a> </p>
+
+                <p>Nothing in this section was against the rules for MPs to accept and they have been correctly declared.</p>
```

Comment:
I wonder if we should make this more prominent just to head off potential complaints.